app/api/events.py

```diff
@@ -168,7 +168,7 @@ def create(self, data, url):
         save_to_db(new_event, "Event saved")
         # set organizer
         role = Role.query.filter_by(name=ORGANIZER).first()
-        uer = UsersEventsRoles(g.user, new_event, role)
+        uer = UsersEventsRoles(user_id=creator_id, event=new_event, role=role)
```

Comment:
The error which we were getting `User instance already exists` was because both were being passed: `g.user.id` [here](https://github.com/fossasia/open-event-orga-server/blob/development/app/api/imports.py#L33) and `g.user` [here](https://github.com/fossasia/open-event-orga-server/blob/development/app/api/events.py#L171). Passing the creator_id that was passed while getting the celery task solves the issue.

I am not much sure about how we can incorporate this without breaking the Event API. Suggestions please!